controllers/defaults/placements.go

```diff
@@ -126,6 +131,30 @@ var (
 				getOcsToleration(),
 			},
 		},
+
+		ProviderServerKey: {
```

Comment:
am not sure its the right place to add these because these are not ocs daemons i.e `DaemonPlacements`

Reply:
We are keeping all the placements here. to align the code and func calls.